packages/shared/src/hooks/useSquadChecklist.tsx

```diff
@@ -35,7 +36,12 @@ type UseSquadChecklist = UseChecklist & {
 const useSquadChecklist = ({
   squad,
 }: UseSquadChecklistProps): UseSquadChecklist => {
-  const { actions, isActionsFetched: isChecklistReady } = useActions();
+  const {
```

Comment:
I'm a bit confused what this has to do with the squad checklist?

Reply:
I'm reusing the same action key (the only related to extension). 
I've checked other usages and changed here because if user is already on extension maybe we don't want to show it as unmarked, not?

Reply:
Wait the current one get's called from squad checklist?
Maybe I just missed that, not sure what it has to do with a squad, but could be right.

Then again why was a change needed then?

Reply:
I dont know why we have a "Download browser extension" step in squad's checklist so since now we are tracking if user is coming from extension I supposed that is a good idea to update this logic too. This is not needed and I can remove this piece of code if blocking 👀 

<img width="1189" alt="image" src="https://github.com/user-attachments/assets/7beed636-7d5c-4fcd-ad06-e902e5dd1bcc" />

Reply:
Rather check with product on this, I don't think it's a decision we can make for now.
Then the whole step is pointless, but it should happen elsewhere for everyone in that case (non experiment based)

Reply:
If working on this right during an experiment is not the right thing to do I'm going to cleanup those changes, just avoiding possible issues! ✅

Reply:
Removed ✔️